modules/appeals_api/spec/requests/v2/supplemental_claims_controller_spec.rb

```diff
@@ -253,12 +253,20 @@ def obj.class
       allow(client_stub).to receive(:upload).and_return(faraday_response)
       allow(faraday_response).to receive(:success?).and_return(true)
 
-      Sidekiq::Testing.inline! do
-        post(path, params: data, headers: headers)
-      end
+      with_settings(Settings.vanotify.services.lighthouse.template_id,
```

Comment:
Same here. There's nothing that's testing the creation of the PDF job.

Reply:
on it